src/main/scala/cromwell/binding/values/WdlFile.scala

```diff
@@ -1,6 +1,7 @@
 package cromwell.binding.values
 
 import cromwell.binding.types.{WdlFileType, WdlType}
+import cromwell.engine.{FileHasher, Hash}
```

Comment:
Can we move Hash and FileHasher into the Binding layer?  The binding layer should not depend at all on the engine layer